core/templates/dev/head/pages/exploration_editor/editor_tab/StateEditor.js

```diff
@@ -168,8 +170,17 @@ oppia.factory('trainingModalService', [
                   editorContextService.getActiveStateName();
                 var state = explorationStatesService.getState(currentStateName);
 
+                // Retrieve the interaction ID.
+                var interactionId = stateInteractionIdService.savedMemento;
+
+                var rulesServiceName = interactionId.charAt(0).toLowerCase() +
+                interactionId.slice(1) + 'RulesService';
```

Comment:
Yep, that sounds good. So, should I create a service that returns the name of rulesService given an interaction ID? Since both - StateResponse and StateEditor would need this service, where should it be created?

PS: Pardon me, I'm still learning Angular :)